assess/events/events.py

```diff
@@ -62,6 +62,17 @@ def events_from_process(cls, process):
         process_exit_dict["value"] = process_exit_dict["tme"] - \
             process_exit_dict["start_tme"]
 
+        if process_dict.get("pid", None) is None:
```

Comment:
Unless you actually have ``None`` values in your dict, you probably want to use this:
```suggestion
        if "pid" not in process_dict:
```